README.md

```diff
@@ -95,4 +95,29 @@ npm run format
 (B) \(-9\)
 (C) \(9\)
 (D) \(27\)
+
+## Alert Example
```

Comment:
Could you please move all custom syntax instructions to the wiki and reference the link here instead? There are expected to be many in the future.

Reply:
Sure! Moved demo content to [wiki](https://github.com/coseeing/Access8MathWeb/wiki/Editor-Custom-Syntax).